tutorials/milvus_lite_arize.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>Would take a second pass to clean up punctuation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>